manifests/g/gdiObjects/xBar/2.0.2111.26/gdiObjects.xBar.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2111.26
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://www.gdiobjects.com/Files/XBInstall.exe
+  InstallerSha256: 3545A81910A18776E9D6BE6E6416E85BDA282CA843A0111AAA76F80B672080AE

```

Comment:
```suggestion
  InstallerSha256: 3545A81910A18776E9D6BE6E6416E85BDA282CA843A0111AAA76F80B672080AE
  ProductCode: '{56E3DC02-59B7-4F96-AF45-F08C001F524E}_is1'
```

Reply:
> 

I don't understand the differences between the InstallerSha256, they seem identical.  As to the ProductCode, that must be an internal reference.  I don't see it referenced in the *.yaml files.  I do have a question regarding the Architecture field.  xBar is a 64-bit  Windows application.  The installer is 32-bit.  Does the 'architecture' field relate to the app or the installer?

Reply:
It is the Product code that I was suggesting be added. Github is just weird about how it displays suggestions

Architecture refers to the application